galley/pkg/runtime/projections/serviceentry/convert/convert_bench_test.go

```diff
@@ -0,0 +1,232 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package convert_test
+
+import (
+	"testing"
+	"time"
+
+	networking "istio.io/api/networking/v1alpha3"
+	"istio.io/istio/galley/pkg/runtime/projections/serviceentry/convert"
+	"istio.io/istio/galley/pkg/runtime/projections/serviceentry/node"
+	"istio.io/istio/galley/pkg/runtime/projections/serviceentry/pod"
+	"istio.io/istio/galley/pkg/runtime/resource"
+
+	coreV1 "k8s.io/api/core/v1"
+)
+
+const (
+	benchNamespace = "benchmarkns"
+)
+
+func BenchmarkService(b *testing.B) {
```

Comment:
A high-level description of the nature of the operation that the benchmark is for, would be useful.